labs/lab9/9.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Lab 9 - Web Application
+layout: lab
+---
+
+## Overview
+
+We’re going to set up a flask app. And we are going to call it basicapp
```

Comment:
Flask should be capitalized

Also I would give it a real name, not just "basicapp." Even something like "lab9" would be better and more descriptive.